meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -4,8 +4,8 @@ DISTRO = "poky-edison"
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-yocto"
 PREFERRED_VERSION_linux-yocto = "4.14.0%"
 PREFERRED_PROVIDER_virtual/bootloader ?=  "u-boot"
-PREFERRED_VERSION_u-boot ?= "edison-v2017.09"
-PREFERRED_VERSION_u-boot-fw-utils ?= "edison-v2017.09"
+PREFERRED_VERSION_u-boot ?= "u-boot-2018.01"
```

Comment:
the value for this one theoretically should be `2018.01`, without `u-boot-` prefix.